### PR TITLE
Automate formula release prep

### PR DESCRIPTION
**Pull request by avocader**
This script is needed by https://jenkins.confluent.io/job/hub-client-prepare-release-brew/ job.
It builds and uploads Confluent Hub client's archive to S3 bucket and pushes branches with the bumped up version of the formula.

### CODE REVIEW

File: release-scripts/prepare-release-formula.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+cd /tmp
+
+rm -rf kafka
+git clone git@github.com:confluentinc/kafka.git
+cd kafka
+git checkout $RELEASE_TAG
+gradle
+./gradlew clients:install connect:api:install connect:runtime:install install_2_11
```
Comment (rhauch) [inline release-scripts/prepare-release-formula.sh]: Why do we need the Connect runtime?

File: release-scripts/prepare-release-formula.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+cd -
+
+for repo in license-file-generator common private-common; do
+	install_mvn_dependency $repo
```
Comment (rhauch) [inline release-scripts/prepare-release-formula.sh]: Is there a reason we can't access Confluent's Maven repo for these dependencies?

**Comment by avocader** [comment]
@rhauch I've updated README.

File: release-scripts/prepare-release-formula.sh (hunk 1 above)
Comment (avocader) [reply release-scripts/prepare-release-formula.sh]: It's a dependency of `license-file-generator`.

File: release-scripts/prepare-release-formula.sh (hunk 2 above)
Comment (avocader) [reply release-scripts/prepare-release-formula.sh]: I wanted this script to be able to release from any tag, even for tags that we don't have Maven artifact published yet.

**Pull request merged**